packages/react-components/lib/tasks/task-table-datagrid.tsx

```diff
@@ -134,11 +135,24 @@ export function TaskDataGridTable({
     {
       field: 'requester',
       headerName: 'Requester',
-      width: 150,
+      minWidth: 160,
       editable: false,
-      valueGetter: (params: GridValueGetterParams) =>
-        params.row.booking.requester ? params.row.booking.requester : 'unknown',
-      flex: 1,
+      renderCell: (cellValues) => {
+        return (
+          <Stack direction="row" alignItems="center" gap={1}>
+            {cellValues.row.booking.requester &&
+            cellValues.row.booking.requester.includes('scheduled') ? (
+              <ScheduleIcon />
```

Comment:
Can we add a tooltip to these icons, such that hovering over them tells us what the icons mean. We can have the tooltips display `User scheduled`, as well as `User submitted` for the case of `UserIcon`.